.buildkite/scripts/benchmark/save-objects/benchmark_objects.ndjson

```diff
@@ -0,0 +1,31 @@
+{"attributes":{"allowHidden":false,"fieldAttrs":"{\"5m_num\":{\"customLabel\":\"\"},\"avg_heap_num\":{\"customLabel\":\"\"},\"avg_non_heap_num\":{\"customLabel\":\"\"},\"max_worker_utilization_num\":{\"customLabel\":\"\"},\"max_worker_concurrency_num\":{\"customLabel\":\"\"},\"max_queue_events\":{\"customLabel\":\"\"},\"max_queue_events_num\":{\"customLabel\":\"\"},\"total_events_out_num\":{\"customLabel\":\"\"},\"avg_cpu_percentage_num\":{\"customLabel\":\"\"},\"release\":{},\"version_num\":{}}","fieldFormatMap":"{\"5m_num\":{\"id\":\"number\",\"params\":{\"pattern\":\"0\"}},\"avg_heap_num\":{\"id\":\"number\"},\"avg_non_heap_num\":{\"id\":\"number\"},\"max_worker_utilization_num\":{\"id\":\"number\"},\"max_worker_concurrency_num\":{\"id\":\"number\"},\"max_queue_events\":{\"id\":\"number\"},\"max_queue_events_num\":{\"id\":\"number\"},\"avg_cpu_percentage_num\":{\"id\":\"number\"}}","fields":"[{\"name\":\"5m_num\",\"script\":\"return Integer.parseInt(doc['max_eps_5m.keyword'].value)\",\"lang\":\"painless\",\"type\":\"number\",\"scripted\":true,\"customLabel\":\"\"}]","name":"benchmark","runtimeFieldMap":"{\"release\":{\"type\":\"boolean\",\"script\":{\"source\":\"def version = doc['version.keyword'].value;\\nif (version.endsWith(\\\"-SNAPSHOT\\\")) {\\n\\temit(false);\\n}\\nelse {\\n\\temit(true);\\n}\"}},\"version_num\":{\"type\":\"long\",\"script\":{\"source\":\"if (doc.containsKey('version.keyword') && !doc['version.keyword'].empty) {\\n    // Get the version string\\n    String version = doc['version.keyword'].value;\\n    \\n    // Remove any suffix after the patch number (e.g., -SNAPSHOT)\\n    int dashIndex = version.indexOf('-');\\n    if (dashIndex != -1) {\\n        version = version.substring(0, dashIndex);\\n    }\\n    \\n    // Find positions of dots\\n    int firstDot = version.indexOf('.');\\n    int secondDot = version.indexOf('.', firstDot + 1);\\n    \\n    // Extract major, minor, and patch parts of the version\\n    int major = Integer.parseInt(version.substring(0, firstDot));\\n    int minor = Integer.parseInt(version.substring(firstDot + 1, secondDot));\\n    int patch = Integer.parseInt(version.substring(secondDot + 1));\\n    \\n    // Combine into a sortable numeric value\\n    emit(major * 10000 + minor * 100 + patch);\\n}\\n\\nemit(0);\"}}}","sourceFilters":"[]","timeFieldName":"timestamp","title":"benchmark_*"},"coreMigrationVersion":"8.8.0","created_at":"2024-09-05T11:52:42.820Z","id":"c5493557-3fd6-4929-8dc2-40d248303942","managed":false,"references":[],"type":"index-pattern","typeMigrationVersion":"8.0.0","updated_at":"2024-09-05T11:52:42.820Z","version":"WzcyNiw3XQ=="}
```

Comment:
It seems we are not installing/using this in the CI. My opinion would be avoiding saving such big objects into the repo. Is there any other way we can think of (like with gist)?

Reply:
This is the dashboard and related charts in Kibana exporting as an save objects. You can import it to your own cluster, so no need to rebuild the graph. It is 128KB, not so big and we need a place to store this object.

Reply:
Yeah 128KB is not big. My point is since we don't use (automatically setup dashboard) in the scripts, if there are other places we can upload and link here.

Reply:
Kibana and ealstic-integration store the save objects in project as well. I think saving it here is straight forward. Maybe I can add instruction on what are the objects included and how to import it.